src/convertToObject.js

```diff
@@ -7,6 +7,15 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((arrKeyAndValue) => arrKeyAndValue.split(':'))
```

Comment:
The variable name 'arrKeyAndValue' is not descriptive enough and does not align with the [CODE STYLE] guideline to avoid namings like 'array', 'object', 'arr', 'obj', 'result'. Consider using a more descriptive name that indicates the content of the variable, such as 'styleDeclaration'.